### ISSUE TITLE
Death knight blood boil damage

### ISSUE THREAD

**Issue by Silverblade1234** [Jihia/Warcraft-5e-Conversion#590]
LordGaster suggests that blood boil damage is too low.

Silverblade: Blood boil is tuned low because (a) it is a bonus action, allowing you to sacrifice little personal damage to add in some aoe damage from a spell, (b) it is selective (no friendly fire) a rarity in aoe spells, and (c) it scales unusually well with upcasting (though this last is really more of a nice perk, and not integral to the spell balance). I think it's where it needs to be, but I could see increasing the damage dice to d8s if other people agree with Gaster that it should be buffed.

**Comment by orjanbp** [comment]
This is probably going to be one of those issues where we should just use it to collect all the feedback that comes in. So let's put it in the 3.1 Watch List, not address it right away, but instead keep an eye on how the feedback plays out?

**Comment by orjanbp** [comment]
Copying in the write-up LordGaster posted as well, so we have that for later.

> Just for BB i have strong feeling that it may sound strong on paper, but with actual numbers, it wouldn't be worth of a spell slot to use on this spell. As for example, lets create ordinary DK in campaign, with perfect conditions for him:
> Lets take average, classical stats - 15 14 13 12 10 8, as they are most popular to use by statistics. Lets imagine that we have a smart player (ye, i know, hard to do that :DDD) and he picked +2 STR +1CHA race, with 15+1 to cha and 14+2 str, giving him +3 and +3 to core stats. 
> Normally, DKs at 4th level would increase STR, as DK is a Martial "Heavy" class. So, we have that statistic : +6(8) to spell attacks and DC 14 for spells. 
> Causal enemy would have +2 - +4 to CON saves (considering that they might have proficiency) at 1-4CR creatures.(even 1/4 goblin got +0 to CON. or 1\8 bandit +1) If we are playing Pre-Cata, quite a lot of people will take Burning Legion or Lich King as BBEG, so quite often creatures will have resistance to at least 1 of 2 damage types of BB. i will take that into account, but will calculate both variant, with res and without.
> Lets also pick up number of players - usually its 3-4, lets take 4 as most popular number. Lets create Average difficulty encounter for then - 5 CR1 creatures. As for non res mobs - Giant Spiders, with +1 to Con - usual enemy for early dungeons. As for res, 5 imps as simple demon-enemy. (i will imagine that imps has only res, not immunity as they have)
> 
> So, you as a DK must spend 1 of 2 spell slots to cast a spell, as a bonus action, which is nice, yes to force GS or Imps to make CON save. +1 leads to result of 13+ will provide with success (40~% chance of success) for both GS and Imps. So, i will take average damage for  both fail, success and fail(res) + success(res)
> 
> At the end:
> Fail - 3.5 + 3.5 = 7 average damage (or 3.5 + 1.7 = 5-6~ damage)
> Success - 1.7 + 1.7 = 3-4 average damage (or 0.85 + 1.7 = 2-3 damage)
> Imagine spending half of your precious recourse to cast a spell that will deal from 3 to 7 damage once.
> and then look at Dragon's Breath spell, that deals 3d6 as bonus action of chosen elemental damage, each round for 1 minute.
> Just my point that - yeah, if it would be a spell for full caster, indeed it is balanced and nice. But it is exclusive for DKs. If you will check Paladin in 5e, there are some gamebreaking spells, if it will be in hands of a full caster, and not paladin (looking at you, bard)

**Comment by Silverblade1234** [comment]
TLDR: Once you consider the action v bonus action difference, you find that shatter and blood boil at 2nd level are equal damage-wise at 3 targets (shatter better at 4+), though there are extra factors to consider. If you change the damage dice to d8s, blood boil is better at 4- targets and shatter better at 5+ targets. But in this case the spell probably scales too well with upcasting, and should be changed to 2d8 fire with +1d8 per spell level. 

Gaster's math is not very useful: obviously any AoE spell is lackluster if used against a single target. Moreover, Gaster continues to ignore the fact that it's a bonus action, so in many cases (2H Blood, petless 2H unholy) this is AoE damage at no action cost to your normal turn.

This is IMO a better way of modeling it versus something like Shatter (or Howling Blast, equivalently): https://docs.google.com/spreadsheets/d/10CDdvPLmhzsUNPQRXzKEh7RwzqWNLFKq2x0QaQfGT50/edit?usp=sharing

This is looking specifically at blood boil at 2nd level, so around DK level 5 where it comes online. So the DK has 2 attacks, and the given hit/save %'s are reasonable at that level. The first column gives number of targets, the second Shatter average total damage, then the current Blood Boil w/ d6 dmg dice AND a full 2 attacks, then a possible Blood Boil w/ d8 dmg dice and attacks.

With d6 blood boil, it+attacks is better than shatter for 1-2 targets, and breaks even (essentially) at 3 targets. At 4+ targets it's better to use your action on Shatter. Do note the other factors in favor of blood boil; it's selective (no friendly fire) and since you still have your attacks, you can for example taunt with dark command + rune strike. If you move blood boil to d8 damage, it moves the break-even point to 4-5 targets (blood boil is slightly better at 4, shatter slightly better at 5).

So this becomes sort of a personal judgment call -- when do you think an actual aoe spell (shatter) should be better at aoe than a hybrid/BA aoe spell (blood boil), that allows you to do some aoe dmg while not significantly impacting your normal turn activities? I like the d6 sweet spot where they're basically even at 3 targets, and at 4+ shatter is better. But others might feel differently!

This is just with the spells at 2nd level; increasing the spell level is going to favor blood boil, because it scales better than shatter. If this is deemed to be a problem, probably the best thing to do is just make blood boil do 2d8 fire, increasing by 1d8 each spell level. This is definitely worth considering if we decide it needs a buff, so that it doesn't end up overpowering other committed aoe spells.

**Comment by Jihia** [comment]
Latest Update: We decided to change the damage of Blood Boil from 1d6 fire damage and 1d6 necrotic damage, plus 1d6 fire and necrotic for each spell level above 2nd. to 2d4 fire and necrotic damage, plus 1d4 fire and necrotic damage for each spell level above 2nd. Issue has been moved to the Watch List for 3.1 and awaiting further playtesting.
